shap/plots/_waterfall.py

```diff
@@ -13,7 +13,7 @@
 # plot that is associated with that feature get overlaid on the plot...it would quickly allow users to answer
 # why a feature is pushing down or up. Perhaps the best way to do this would be with an ICE plot hanging off
 # of the bar...
-def waterfall(shap_values, max_display=10, show=True):
+def waterfall(shap_values, max_display=10, ax=None, show=True):
```

Comment:
this is technically a breaking change. Could you pls set `ax` as the last argument to prevent the breaking change? I understand that this might be handy in some cases to give the user more control, but could you pls elaborate a bit on the rational?